### PR TITLE
Improvements and licence

### PR DESCRIPTION
**Pull request by scoopex**
Thanks for publishing this tool at github.

This pull requests contains the following changes:
- port to python3
- more relaxed log parsing
- allow one ore more logfiles as arguments
- add the BSD-2 license 

Note about the license introduction:

The original release, published by you, did not contain a license. Therefore i added the very relaxed BSD-2 license which helps you and my organization to prevent problems with users of this software.  We would like to have a clear license state for this tool to use it for internal usage.

If you agree the license change proposal, i would be happy if just accept this pull request.
Otherwise it would be great if your submit your licensing conditions or a license description as part of  this project.

### CODE REVIEW

**Comment by dfdeshom** [comment]
Hi @scoopex , the changes look great to me. Can you add my name to the copyright in the license? Aside from that, the BSD license looks fine to me. Thanks!

File: README.md [hunk 1]
```diff
@@ -68,4 +68,11 @@ Notes and Copyright
...
 The tool was forked from [solr-loganalyzer](https://github.com/dfdesho/solr-loganalyzer).
 
-This tool is distributed with the [BSD-2 Licence](LICENCE).
\ No newline at end of file
+This tool is distributed with the [BSD-2 Licence](LICENCE).
+
+Authors
+=======
+
+* Martin Filliau <dfdeshom@gmail.com>
```
Comment (dfdeshom) [inline README.md]: My name is Didier Deshommes :)

Comment (scoopex) [reply README.md]: I am sorry, i accidentally picked up the wrong name :-)

**Comment by dfdeshom** [comment]
Thanks!

**Pull request merged**